### PR TITLE
API v3 and Python 3

### PR DESCRIPTION
**Pull request by Ondkloss**
Moved to the Have I Been Pwned v3 API, which does require an API-key for some of its actions. For example you must private an API-key to check if an account has been breached. With these commits you can provide the API-key either through the `--api_key` command line argument or set it in the `HIBP_API_KEY` environment variable.

Have also adjusted code for Python 3, mostly by using `print('abc')` instead of `print abc`.

The code has also been slightly formatted. Some adjustments to reported errors. There are no actual adjustments to the `--passwd` command line argument.

### CODE REVIEW

File: pwnd_checker/main.py [hunk 1]
```diff
@@ -2,102 +2,126 @@
...
-      json_response = json.loads(response_json.text)
-      for response in json_response:
-         print "1. Website Name: {0}".format(response['Name'])
-         print "2. Domain: {0}".format(response['Domain'])
-         print "3. Breach date: {0}\n".format(response['BreachDate'])
-   except Exception as e:
-      click.secho(str(e), fg="red", bold=True)
-
-#class to handle optional args like: passwd
+url = 'https://haveibeenpwned.com/api/v3/'
```
Comment (lucifercr07) [inline pwnd_checker/main.py]: Shall we keep the `v3` part as api_version var so that can be configurable?

File: pwnd_checker/main.py [hunk 2]
```diff
@@ -2,102 +2,126 @@
...
+
+def get_headers(api_key):
+    return {
+    "hibp-api-key": api_key,
+    'user-agent': 'pwnd_checker',
+}
+
+def get_breachedaccount_response(path, headers):
```
Comment (lucifercr07) [inline pwnd_checker/main.py]: `get_breachedaccount_response` -> `get_breached_account_response`

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
Please submit output in commit summary as `Testing done:` before submitting.

**Comment by Ondkloss** [comment]
Pushed some changes based on your review @lucifercr07.

**Comment by lucifercr07** [comment]
Would you also like to publish v3 of this on `pip` ? Please add yourself as co-author for module.

**Pull request merged**